### PR TITLE
chore: update vue to latest version of vue2

### PR DESCRIPTION
**Pull request by eddieferrer**
Upgrade vue to latest 2.x version. 

Tons of bug fixes in the release notes, and most of the vue3 features backported. 

There is a warning: "[@vue/compiler-sfc] ::v-deep usage as a combinator has been deprecated. Use :deep(<inner-selector>) instead." on build. I can clean that  up in a followup PR

### CODE REVIEW

**Review by dyersituations — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -53,7 +53,6 @@
 		"@mdi/js": "^5.9.55",
 		"@sentry/tracing": "^6.19.6",
 		"@sentry/vue": "^6.19.6",
-		"@vue/composition-api": "^1.4.6",
```
Comment (mcstover) [inline package.json:56]: Is this now included by default?

File: src/main.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /* global UI_TAG */
 /* eslint-disable vue/multi-word-component-names */
-import Vue from 'vue';
-import VueCompositionAPI from '@vue/composition-api';
+import { Vue, VueCompositionAPI } from 'vue';
```
Comment (mcstover) [inline src/main.js:3]: yes it is~

**Comment by mcstover** [comment]
@eddieferrer Does `vue-server-renderer` also have a corresponding update?

**Comment by eddieferrer** [comment]
I can't merge this in because of our unit test. 

vue-test-utils: https://github.com/vuejs/vue-test-utils/issues/1983
vue-jest does not support vue 2.7, see: https://github.com/vuejs/vue-jest/pull/489

**Comment by eddieferrer** [comment]
> @eddieferrer Does `vue-server-renderer` also have a corresponding update?

Yep, added

**Comment by eddieferrer** [comment]
Closing this, see links above ^

**Comment by mcstover** [comment]
Maybe they'll get it cleaned up in a week or so. There had been at least some activity there

**Pull request closed without merging**